test/integration/framework/src/siftool/run_env.py

```diff
@@ -605,6 +605,11 @@ def run(self):
         # This script is also called from tests
 
         relayer_db_path = os.path.join(self.test_integration_dir, "sifchainrelayerdb")
+
+        # Prevent starting over dirty/existing relayer_db_path
+        if self.cmd.exists(relayer_db_path):
```

Comment:
I don't think relayer db exists in peggy2, is this code still necessary?

Reply:
Siftool supports peggy1 and peggy2. This path is part of peggy1, and I lost many hours because I forgot to clean it and things didn't work as expected. This warning is here to prevent people from losing more time, and it will be "garbage collected" when peggy1 is EOL.